### PR TITLE
cfr viewer polish

### PR DESCRIPTION
**Pull request by donjo**
A few changes to make sure this looks right on the CFR viewer and some polish on non-viewer parts.

Note: I added a class for the ToC on the CFR tab so it could be styled the way it is currently on master while allowing the cfr viewer navigation to look correct. This process also cleaned up some random `<li>`s on the cfr viewer nav having double-thick border lines when they shouldn't have.

I think this may be everything that has to be complete before I can start safely renaming color  and font variables so let me know if there's something I overlooked before I proceed. 

screen shot 2016-06-24 at 5 06 15 pm

### CODE REVIEW

File: regulations/static/regulations/css/less/module/drawer-content.less [hunk 1]
```diff
@@ -89,8 +97,10 @@ This contains all of the styles specific to the TOC
 
   a {
     display: block;
-    padding: 7px 15px 8px 5px;
+    padding: 7px 15px 8px 105px;
+    margin-left: -100px;
     line-height: 1.3;
+    outline: none;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/drawer-content.less]: Per discussion in chat, we should keep the outline for accessibility.

**Comment by cmc333333** [comment]
Minor, and shouldn't hold back this issue, but probably holds back the "complete baseline": Second tab of CFR viewer has a non-blue header:

screen shot 2016-06-28 at 12 14 12 pm
screen shot 2016-06-28 at 12 15 47 pm

**Comment by cmc333333** [comment]
Looks good! Works for normal CFR viewing as well as diff view.

CC @adborden as he's worked on those height variables

**Comment by adborden** [comment]
+1

**Pull request merged**